src/hooks/useLoggedInUser.tsx

```diff
@@ -8,11 +8,20 @@ const UserContext = createContext<User | undefined>(undefined);
 
 export const UserProvider: FC<PropsWithChildren> = ({ children }) => {
 	// Hold user info in state
-	const [user, setUser] = useState<User>();
+	const [user, setUser] = useState<User>(
+		localStorage.getItem('user')
+			? JSON.parse(localStorage.getItem('user'))
```

Comment:
Taky ti to tady IDE podtrhává?

```
Argument of type 'string | null' is not assignable to parameter of type 'string'.
  Type 'null' is not assignable to type 'string'
```

Reply:
Problém je podle mě to, že čteš dvakrát z localStorage, přičemž není zaručené, že mezitím nedojde ke změně výsledku. Opraveno:

`const userFromStorage = localStorage.getItem('user');
	const [user, setUser] = useState<User | undefined>(
		userFromStorage ? JSON.parse(userFromStorage) : undefined
	);`

Reply:
Make sense!